### PR TITLE
follow: init at 0.0.1-alpha.17

### PR DESCRIPTION
**Pull request by iosmanthus**
## Description of changes
pack https://github.com/RSSNext/Follow.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by sifmelcara** [comment]
Is it possible to build from source?

**Comment by iosmanthus** [comment]
> Is it possible to build from source?

It's possible, however, that it might be the simplest method to wrap the upstream AppImage release.

**Comment by iosmanthus** [comment]
@bobby285271 PTAL :heart:

**Comment by iosmanthus** [comment]
> Is it possible to build from source?

Done

File: pkgs/by-name/fo/follow/package.nix [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  fetchFromGitHub,
+  imagemagick,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  nodejs,
+  pnpm,
+  stdenv,
+}:
+let
```
Comment (NickCao) [inline pkgs/by-name/fo/follow/package.nix:12]: This is purely aesthetic, but we'd better just drop the let in block.

File: pkgs/by-name/fo/follow/package.nix [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    hash = "sha256-K8IM2kE7qhEBux4eta1ma/timSeljzf0MbOUeJ4JCIc=";
+  };
+  env = {
+    ELECTRON_SKIP_BINARY_DOWNLOAD = "1";
+    VITE_WEB_URL = "https://app.follow.is";
+    VITE_API_URL = "https://api.follow.is";
+    VITE_IMGPROXY_URL = "https://thumbor.follow.is";
+    VITE_SENTRY_DSN = "https://e5bccf7428aa4e881ed5cb713fdff181@o4507542488023040.ingest.us.sentry.io/4507570439979008";
+    VITE_BUILD_TYPE = "production";
+    VITE_POSTHOG_KEY = "phc_EZGEvBt830JgBHTiwpHqJAEbWnbv63m5UpreojwEWNL";
```
Comment (NickCao) [inline pkgs/by-name/fo/follow/package.nix:32]: Please add a link pointing to where these *magic* variables are from.

**Review by NickCao — COMMENTED** [review]
Overall looks pretty good!

File: pkgs/by-name/fo/follow/package.nix (hunk 1 above)
Comment (iosmanthus) [reply pkgs/by-name/fo/follow/package.nix:12]: Roger.

File: pkgs/by-name/fo/follow/package.nix (hunk 2 above)
Comment (iosmanthus) [reply pkgs/by-name/fo/follow/package.nix:32]: OK, these magic vars are copy from the upstream release workflow.

**Comment by iosmanthus** [comment]
@NickCao comment addressed, PTAL.

**Review by NickCao — APPROVED** [review]
Please squash everything into two commits, and we are good to go!

1. maintainers: add iosmanthus
2. follow: init at 0.0.1-alpha.13

**Comment by iosmanthus** [comment]
@NickCao Done, PTAL :heart:

**Comment by NickCao** [comment]
@ofborg build follow

**Review by NickCao — APPROVED** [review]

**Pull request merged**